frontend/src/app/auth/guards/auth-guard.service.ts

```diff
@@ -9,10 +9,10 @@ export class AuthGuardService implements CanActivate {
   constructor(private authService: AuthService, private router: Router) { }
 
     canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): boolean {
-        if (this.authService.isLoggedIn()) {
-            return true;
+        if (!this.authService.isLoggedIn()) {
+            this.router.navigate(['login']);
+            return false;
         }
-        this.router.navigate(['/login']);
-        return false;
+        return true;
```

Comment:
The two return statements could be replaced with `return this.authService.isLoggedIn()` outside of the if statement.